### PR TITLE
Added support for Quandify CubicMeter 1.1

### PR DESCRIPTION
**Pull request by peter-quandify**

### CODE REVIEW

**Comment by peter-quandify** [comment]
Ready for merge

File: vendors/axioma/drivers/w1/package-lock.json
Comment (tomHallActility) [inline vendors/axioma/drivers/w1/package-lock.json:1]: Could you discard your changes from this file please? Thanks.

**Review by tomHallActility — CHANGES_REQUESTED** [review]
Hi Peter,

Thank you for making this Pull Request.
The CubicMeter is in now in two categories, right?
I would suggest instead of deleting the original one, we could just change its name to cubicmeter 1.1 copper and add a new model for the plastic one, to avoid complexifying deployments on our catalogs.
Is that ok with you?

Once that is done, I will merge your PR and make a few adaptations on our side since we'll have all the information we need, and your changes will then be integrated to the next deployments.

Best regards,
Tom

**Comment by peter-quandify** [comment]
Hi Tom,
I changed the modelId in cubicmeter-1.1-copper/model.yml to match the one that was in the cubicmeter folder before.
Was this how you intended the change?

Also removed the axioma from the PR, sorry for that random mistake.

**Comment by tomHallActility** [comment]
Ok thank you very much.
I'll go ahead and merge that then make a few changes on my part.

**Review by tomHallActility — APPROVED** [review]

**Pull request merged**